python/pfs/drp/stella/apertureCorrections.py

```diff
@@ -41,7 +41,7 @@ class MeasureApertureCorrectionsConfig(Config):
     def setDefaults(self):
         super().setDefaults()
         self.apertureFlux.radii = [3.0]
-        self.fit.mask = ReferenceLineStatus.getMasks().flags.keys()
+        self.fit.mask = ["NOT_VISIBLE", "BLEND", "SUSPECT", "REJECTED", "BROAD"]
```

Comment:
Shouldn't the ReferenceLineStatus bit mask names be used here rather than literal strings?

Reply:
No, that's [defined](https://github.com/Subaru-PFS/drp_stella/blob/3a2c02d41eea00b16afb3649f0ab3fd04e5df1bd/python/pfs/drp/stella/fitFocalPlane.py#L22-L23) as a list of strings that get converted to a bit mask [here](https://github.com/Subaru-PFS/drp_stella/blob/3a2c02d41eea00b16afb3649f0ab3fd04e5df1bd/python/pfs/drp/stella/fitFocalPlane.py#L90).